plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/AnalyzeHandle.java

```diff
@@ -28,17 +28,20 @@
     private final boolean initialAnalyze;
     private final Optional<Instant> filesModifiedAfter;
     private final Optional<Set<String>> columns;
+    private final Optional<Boolean> recalculateAllStatistics;
```

Comment:
Does this need to be Optional? Looks like it just always defaults to `false`?

Reply:
removed